alibi_detect/od/ensemble.py

```diff
@@ -0,0 +1,26 @@
+import imp
+from typing import Optional, List
+from alibi_detect.od.base import OutlierDetector
+from alibi_detect.od.aggregation import BaseTransform, AverageAggregator, PValNormaliser
+from alibi_detect.od.processor import BaseProcessor
+
+
+class Ensemble(OutlierDetector):
+    def __init__(
+            self,
+            detectors: List[OutlierDetector], 
+            aggregator: Optional[BaseTransform] = AverageAggregator(), 
```

Comment:
A good default aggregator is likely quite important for the ensemble. Simply averaging has its drawbacks.